defaults/main.yml

```diff
@@ -102,6 +102,14 @@ jenkins_secret_files: []
 # can be useful to see which plugins changed during a deployment.
 jenkins_show_plugins_yaml_changes: true
 
+# How long to wait between retries when checking if Jenkins has started up. Using lower
+# values will reduce latency in the playbook at the expense of log spam.
+jenkins_startup_retry_delay_sec: 5
```

Comment:
Do we have any insights on the startup time of our Jenkins hosts, e.g. a rough average?  Let's say if our instances usually take 60s to start, then we should this delay half of that time in my opinion.

Reply:
Just one thing, we should state that this must be non-zero otherwise calculating `retries` will fail.

Reply:
Let's assert this in `sanity-check.yml`. See f929449.